content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -99,7 +99,7 @@ If it is, Topology Manager will store this and the *Hint Providers* can then use
 resource allocation decision.
 If, however, this is not possible then the Topology Manager will reject the pod from the node. This will result in a pod in a `Terminated` state with a pod admission failure.
 
-Once the pod is in a `Terminated` state, the Kubernetes scheduler will **not** attempt to reschedule the pod. It is recommended a Deployment with Replicas to trigger a redeploy of the pod.
+Once the pod is in a `Terminated` state, the Kubernetes scheduler will **not** attempt to reschedule the pod. It is recommended to use a Deployment with Replicas to trigger a redeploy of the pod.

```

Comment:
@sharjeelaziz , should the text be identical to line 87 or something similar to line 102?

`It is recommended to use a ReplicaSet or Deployment to trigger a redeploy of the pod`
OR
`It is recommended to use a Deployment with replicas to trigger a redeploy of the Pod.`

https://deploy-preview-18201--kubernetes-io-master-staging.netlify.com/docs/concepts/workloads/controllers/replicaset/#deployment-recommeded

Reply:
@kbhawkey my intention was to fix the typo in the document and keep the message given by the original author. Also at this point I cannot say with enough confidence whether the two statements should be the same as I think maybe first statement might not be applicable in this case.

Reply:
OK @sharjeelaziz . I suggest these changes to conform to the k8s docs style guide:
substitute replicas for Replicas
substitute Pod for pod

Reply:
Thanks @kbhawkey . I have made the changes. Sorry for not understanding your suggested changes earlier.